### PR TITLE
PHP Site Deployment

### PR DESCRIPTION
**Pull request by Benjiiim**
Now that kudu container on Azure App Service on Linux has PHP and composer support (cf https://github.com/Azure-App-Service/kudu/pull/15), it seems a great time for kudu to be able to detect php apps.
PHP deployement logic has been submitted on https://github.com/projectkudu/KuduScript/pull/69
This PR was inspired from the Ruby support at #2268

### CODE REVIEW

**Comment by nickwalkmsft** [comment]
KuduScript changes have been merged, if you can modify updateNodeModules, we should be able to get this in as well.

**Comment by Benjiiim** [comment]
updateNodeModules updated to get the last version of KuduScript which include the PHP deployment script.

File: Kudu.Core/Deployment/Generator/SiteBuilderFactory.cs [hunk 1]
```diff
@@ -159,6 +159,10 @@ private ISiteBuilder ResolveNonAspProject(string repositoryRoot, string projectP
             {
                 return new RubySiteBuilder(_environment, perDeploymentSettings, _propertyProvider, repositoryRoot, projectPath);
             }
+            else if (IsPHPSite(sourceProjectPath))
```
Comment (davidebbo) [inline Kudu.Core/Deployment/Generator/SiteBuilderFactory.cs]: I think we should only do this for Linux, or we might break existing PHP sites on Windows.

Comment (davidebbo) [inline Kudu.Core/Deployment/Generator/SiteBuilderFactory.cs]: Oh, I see that the enabler class checks for this, so we're good.

**Pull request merged**